charts/knuto/values.yaml

```diff
@@ -1,19 +1,32 @@
 # image -- Which knuto docker image to install
 image: niradynamics/knuto:a1d35fb
```

Comment:
This version will have to change but I cannot push to docker before I have merged this review